lib/azkaban_scheduler/session.rb

```diff
@@ -5,9 +5,9 @@ module AzkabanScheduler
   class Session
     attr_accessor :id
 
-    def self.start(client, username, password, headers=nil)
+    def self.start(client, username, password)
       params = {'action' => 'login', 'username' => username, 'password' => password}
-      response = client.post('/', params, headers)
+      response = client.post('/', params)
```

Comment:
Here you still need to pass the headers, otherwise communication through oauth tunnel might not work.

Reply:
The headers are on the client so they will be passed into post before it makes the request.